### PR TITLE
Updates from adam's review

### PR DESCRIPTION
**Pull request by dericed**
Updates that respond to https://mailarchive.ietf.org/arch/msg/cellar/yvXmJPWGkUISXbdt0aZ-UkC57I8

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -1121,29 +1121,31 @@ Element IDs are described in section Element ID. Element IDs are encoded using t
 
 This IANA Registry only applies to Elements that can be contained in the EBML Header, thus including Global Elements. Elements only found in the EBML Body have their own set of independent Element IDs and are not part of this IANA Registry.
 
-The VINT Data value of one-octet Element IDs MUST be between 0x01 and 0x7E. These items are valuable because they are short, and need to be used for commonly repeated elements. Values from 1 to 126 are to be allocated according to the "RFC Required" policy [@!RFC8126].
+One-octet Element IDs MUST be between 0x81 and 0xFE. These items are valuable because they are short, and need to be used for commonly repeated elements. Element IDs are to be allocated within this range according to the "RFC Required" policy [@!RFC8126].
 
-The VINT Data value of two-octet Element IDs MUST be between 0x007F and 0x3FFE. Numbers are to be allocated within this range according to the "Specification Required" policy [@!RFC8126].
+The following one-octet Element ID is RESERVED: 0x80.
```
Comment (adamroach) [inline specification.markdown]: Shouldn't this also list 0xFF? The second-to-last line in Table 3 would seem to put it in a special category (like the other all-consecutive-one bit patterns listed in the paragraphs below).

File: specification.markdown [hunk 2]
```diff
@@ -239,7 +239,7 @@ An Unsigned Integer Element stores an integer (meaning that it can be written wi
 
 A Float Element MUST declare a length of either zero octet (0 bit), four octets (32 bit) or eight octets (64 bit). If the EBML Element is not defined to have a default value, then a Float Element with a zero-octet length represents a numerical value of zero.
 
-A Float Element stores a floating-point number as defined in [@!IEEE.754.1985].
+A Float Element stores a floating-point number in binary interchange format as defined in [@!IEEE.754.1985].
```
Comment (robUx4) [inline specification.markdown]: Should probably mention it's the 32-bit and 64-bit interchange formats.

File: specification.markdown [hunk 3]
```diff
@@ -1121,29 +1121,39 @@ Element IDs are described in section Element ID. Element IDs are encoded using t
 
 This IANA Registry only applies to Elements that can be contained in the EBML Header, thus including Global Elements. Elements only found in the EBML Body have their own set of independent Element IDs and are not part of this IANA Registry.
 
-The VINT Data value of one-octet Element IDs MUST be between 0x01 and 0x7E. These items are valuable because they are short, and need to be used for commonly repeated elements. Values from 1 to 126 are to be allocated according to the "RFC Required" policy [@!RFC8126].
+One-octet Element IDs MUST be between 0x81 and 0xFE. These items are valuable because they are short, and need to be used for commonly repeated elements. Element IDs are to be allocated within this range according to the "RFC Required" policy [@!RFC8126].
 
-The VINT Data value of two-octet Element IDs MUST be between 0x007F and 0x3FFE. Numbers are to be allocated within this range according to the "Specification Required" policy [@!RFC8126].
+The following one-octet Element ID is RESERVED: 0xFF and 0x80.
```
Comment (robUx4) [inline specification.markdown]: plural ? (IDs are reserved)

File: specification.markdown [hunk 4]
```diff
@@ -1121,29 +1121,39 @@ Element IDs are described in section Element ID. Element IDs are encoded using t
...
 
-The VINT Data value of three-octet Element IDs MUST be between 0x4001 and 0x1FFFFE. Numbers may be allocated within this range according to the "First Come First Served" policy [@!RFC8126].
+Two-octet Element IDs MUST be between 0x407F and 0x7FFE. Element IDs are to be allocated within this range according to the "Specification Required" policy [@!RFC8126].
 
-The numbers 0x1FFFFF and 0x200000 are RESERVED.
+The following two-octet Element IDs are RESERVED: 0x7FFF and 0x4000.
 
-Four-octet Element IDs are numbers between 0x101FFFFF and 0x1FFFFFFE. Four-octet Element IDs are somewhat special in that they are useful for resynchronizing to major structures in the event of data corruption or loss. As such four-octet Element IDs are split into two categories. Four-octet Element IDs whose lower three octets (as encoded) would make printable 7-bit ASCII values (0x20 to 0x7E, inclusive) MUST be allocated by the "Specification Required" policy. Sequential allocation of values is not required: specifications SHOULD include a specific request, and are encouraged to do early allocations.
+The two-octet ranges of 0x0000 to 0x3FFFF and 0x8000 to 0xFFFF are not valid for use as an Element ID.
```
Comment (robUx4) [inline specification.markdown]: 0x3FFF not 0x3FFFF

**Comment by dericed** [comment]
updated to @robUx4's comments, ready for re-review

**Comment by dericed** [comment]
Nudging for a review as merging this before writing the subsequent pr's for the other discusses would be helpful.

**Comment by dericed** [comment]
thanks @robUx4 & @adamroach

**Pull request merged**